subprojects/platform-jvm/src/main/java/org/gradle/jvm/tasks/Jar.java

```diff
@@ -92,6 +98,16 @@ public void execute(FileCopyDetails details) {
         });
     }
 
+    @Override
+    protected CopyAction createCopyAction() {
```

Comment:
Shouldn't we have a protected `createCopyAction(Comparator)` method in `AbstractArchiveTask` so we could call this one instead of duplicating all the code?